js/app.js

```diff
@@ -21,6 +21,9 @@ const defaultModules = require(`${__dirname}/../modules/default/defaultmodules`)
 global.version = require(`${__dirname}/../package.json`).version;
 Log.log(`Starting MagicMirror: v${global.version}`);
 
+// Log system information.
+Utils.logSystemInformation();
```

Comment:
I think putting it in Utils is ok for now. Moving it to somewhere else if someone has a better idea is always possible